meta-mel/conf/distro/mel.conf

```diff
@@ -297,7 +297,7 @@ SDK_NAME_PREFIX = "${DISTRO}"
 
 # Adjust installer name and default install path
 SDK_NAME = "${SDK_NAME_PREFIX}-${SDK_VERSION}-${IMAGE_BASENAME}-${MACHINE}"
-SDKPATHINSTALL = "/opt/${DISTRO}/${SDK_VERSION}/sdk/${IMAGE_BASENAME}-${MACHINE}"
+SDKPATHINSTALL = "~/${DISTRO}/sdk/${SDK_VERSION}/${IMAGE_BASENAME}-${MACHINE}"
```

Comment:
what happens if it is a server or something similar where a homedir might not be set? This is just the default right?

Reply:
Yeah, just the default, it prompts for the path when running the installer.